python_bindings/README.md

```diff
@@ -0,0 +1,33 @@
+# A python wrapper for easi
+
+Install with `-DPYTHON_BINDINGS=ON` cmake option.
```

Comment:
Note that ASAGI and ImpalaJIT have to be compiled with `cmake .. -DCMAKE_CXX_FLAGS="-fPIC"`.